pkg/preflights/host-preflight.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: troubleshoot.sh/v1beta2
```

Comment:
This is a subset of the host support bundle specs. Not all apply to host preflights.

Reply:
Not sure, should we use the `uri` parameter here ?

Reply:
I would think so, I'm not aware of any reason to intentionally not use the uri

Reply:
There is one use case to consider here though. If a new analyser/collector is built and added to troubleshoot, then newer versions of EC adopt it in their default host preflight specs, there is a risk of older install/restore/upgrade/join preflights failing. This is because `preflight` command reports failures if an analyser is unknown.

Reply:
I would be inclined to call that a bug and fix it. URI should work for this use case and this seems like a blocker to using it.